libvast/include/vast/concept/parseable/vast/data.hpp

```diff
@@ -75,9 +61,9 @@ struct data_parser : parser_base<data_parser> {
       | parsers::boolean
       | parsers::qqstr
       | parsers::pattern
-      | '[' >> ~(x % ',') >> trailing_comma >> ']'
+      | parsers::list
       | '{' >> ~as<map>(kvp % ',') >> trailing_comma >> '}'
```

Comment:
Should it be removed, while i am at it? I think there is no need for this code to support parsing maps as they map value cannot be used anyways.

Reply:
Nothing should break as part of it, so I'd say yes.